### PR TITLE
Work for aviate catalog plugin

### PR DESCRIPTION
**Pull request by reshmabidikar**

### CODE REVIEW

File: userguide/aviate/aviate-catalog.adoc [hunk 1]
```diff
@@ -3,3 +3,59 @@
 include::{sourcedir}/aviate/includes/aviate-card.adoc[]
 
 == Introduction
+
+The Aviate plugin can be used as a catalog plugin. The Aviate catalog plugin serves as an alternative to the XML catalog managed within Kill Bill core. It provides APIs that operate at the plan/product/pricelist level, offering greater flexibility and granularity compared to the catalog version level. These APIs enable users to create individual catalog entries such as plans, products, and pricelists without the need to manage entire catalog versions. The catalog data created through the plugin is stored in tables managed by the Aviate plugin.
+
+When the aviate catalog plugin features are enabled, the catalog served by the plugin takes priority over any _existing catalog entries_ created from regular KB catalog APIs (`/1.0/kb/catalog`) documented https://killbill.github.io/slate/catalog.html#upload-a-catalog-as-xml[here].
```
Comment (pierre) [inline userguide/aviate/aviate-catalog.adoc]: Nit: apidocs.killbill.io

File: userguide/aviate/aviate-catalog.adoc [hunk 2]
```diff
@@ -3,3 +3,59 @@
...
+
+The Aviate plugin can be installed as documented in the [How to Install the Aviate Plugin] doc.
+
+=== Enabling Catalog APIs
+
+To use the catalog API capabilities provided by the Aviate plugin, ensure that KB is started with the following properties:
+
+```
+com.killbill.billing.plugin.aviate.enableCatalogApis=true
+com.killbill.billing.plugin.aviate.enableHealthReporter=false //TODO is this required?
```
Comment (pierre) [inline userguide/aviate/aviate-catalog.adoc]: Nope, both properties shouldn't be required.

File: userguide/aviate/how-to-install-the-aviate-plugin.adoc [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+* Ensure that you have https://curl.haxx.se/[_cURL_] installed. If you are on Windows, we recommend that you use https://git-scm.com/download/win[_Git Bash_] to run the `cURL` commands.
+
+== Plugin Installation
+
+The Aviate plugin is not an open-sourced plugin and hence cannot be installed using the standard https://docs.killbill.io/latest/plugin_installation#_plugin_installation[__plugin installation steps__].
+
+This section lists the steps to be followed for the Aviate plugin installation.
+
+//TODO - Steps to create the DB tables are not mentioned here, I'm assuming that com.killbill.billing.plugin.aviate.enableMigrations=true will automatically create the tables?
```
Comment (pierre) [inline userguide/aviate/how-to-install-the-aviate-plugin.adoc:17]: Yes, they should. We should explain how the flyway migration process works and also share the DDL somewhere (https://docs.killbill.io/latest/aviate-mysql.ddl ?)

**Review by pierre — APPROVED** [review]

File: userguide/aviate/how-to-install-the-aviate-plugin.adoc (hunk 3 above)
Comment (reshmabidikar) [reply userguide/aviate/how-to-install-the-aviate-plugin.adoc:17]: I'll gather more information about this and address this in a subsequent PR.

**Pull request merged**